### PR TITLE
Jacobgoldfarb

### PR DESCRIPTION
**Pull request by jacobgoldfarb**
Got rid of some annoying compile time errors by using optional binding.

### CODE REVIEW

File: CardParts/src/Classes/Card Parts/CardPartView.swift [hunk 1]
```diff
@@ -20,15 +20,11 @@ public protocol CardPartView {
 extension CardPartView {
     
     public var view: UIView! {
-        get {
-            return self as! UIView
-        }
+        return (self as! UIView)
```
Comment (bharathmurs) [inline CardParts/src/Classes/Card Parts/CardPartView.swift]: I would get rid of implicitly unwrapped optional and have something like this.

```
public protocol CardPartView {
	...    
        var view : UIView { get }
	...
}
```

and

```
extension CardPartView where Self: UIView {   
    public var view: UIView {
            return self
    }
}
```

File: CardParts/src/Classes/CardPartTableViewCardPartsCell.swift [hunk 2]
```diff
@@ -79,8 +79,9 @@ open class CardPartTableViewCardPartsCell : UITableViewCell {
 			}
 			
 			let metrics = ["leftMargin" : cardPart.margins.left - 28, "rightMargin" : cardPart.margins.right - 28]
-			stackView.addConstraints(NSLayoutConstraint.constraints(withVisualFormat: "H:|-leftMargin-[cardPartView]-rightMargin-|", options: [], metrics: metrics, views: ["cardPartView" : cardPart.view]))
-			
+            if let cardPartView = cardPart.view {
```
Comment (bharathmurs) [inline CardParts/src/Classes/CardPartTableViewCardPartsCell.swift]: By making the above suggested changes, you don't have to do `if-lets` anymore.

File: CardParts/src/Classes/Card Parts/CardPartView.swift (hunk 1 above)
Comment (bharathmurs) [reply CardParts/src/Classes/Card Parts/CardPartView.swift]: Using implicitly unwrapped optionals and force casting would be crash-prone.

**Comment by croossin** [comment]
@jacobgoldfarb Any updates on this PR?

**Comment by jacobgoldfarb** [comment]
> @jacobgoldfarb Any updates on this PR?

Will update after work today

**Comment by croossin** [comment]
@jacobgoldfarb How is the PR coming?

**Comment by croossin** [comment]
I will be closing this PR.  Feel free to open up again if you get back to it @jacobgoldfarb

**Pull request closed without merging**